app/services/icon_mapping_helper.rb

```diff
@@ -0,0 +1,32 @@
+module IconMappingHelper
```

Comment:
Can you make the title include `Catalog`?  I don't think this set of terms will match for other sources.

Reply:
I'm shipping this to get the right icons for review, but I think I agree this mapping is a little confusing. It might be clearer if there were separate maps for catalog (whose terms we control) and articles+?